contracts/token/IMinter.sol

```diff
@@ -29,9 +29,9 @@ interface IMinter {
 
     function inflation() external view returns (uint256);
 
-    function maxInflation() external view returns (uint256);
+    function inflationCeiling() external view returns (uint256);
 
-    function minInflation() external view returns (uint256);
+    function inflationFloor() external view returns (uint256);
```

Comment:
I like the rename! Make sure there's no occurences of `[Mm]ax.+[Ii]nflation` on the project anymore 👀